packages/ic-http-certification/src/lib.rs

```diff
@@ -0,0 +1,242 @@
+//! # Internet Computer HTTP Certification
```

Comment:
Unfortunately this is the same as `README.md`, but with links added to items so it will navigate around docs.rs in a nice way. I'm a bit conflicted by duplicating this, but it ensures that everywhere a developer encounters the library, they will have documentation to greet them. docs.rs doesn't have an option to include a readme :(